XbSymbolDatabase.c

```diff
@@ -417,6 +417,24 @@ bool CompareOOVPAToAddress(OOVPA *Oovpa, memptr_t cur, uintptr_t xb_start_virt_a
     return true;
 }
 
+// Return if the given (XRef'erenced) address is not set yet.
+inline bool IsXRefAddrUnset(xbaddr XRefAddr)
```

Comment:
Before I forget, include `static` before inline since it's not used outside of this file.